effectful/handlers/llm/sampling.py

```diff
@@ -45,3 +47,55 @@ def n_votes_ahead():
                 tasks.append(executor.submit(interpreter(intp)(fwd), *args, **kwargs))
         executor.shutdown()
         return self.votes.most_common(1)[0][0]
+
+
+def sample(template, n):
```

Comment:
shouldn't sample have a type signature? or is this the issue from before with mypy?

Reply:
Ok, done.